_rules/inline-link-in-paragraph-has-distinguishable-content-e7cc2b.md

```diff
@@ -0,0 +1,142 @@
+---
+id: e7cc2b
+name: Inline link has distinguishable content
+rule_type: atomic
+description: |
+  This rule checks that inline links have content that distinguishes them from the surrounding text
+accessibility_requirements:
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [visible][] HTML element that is a [semantic link][], for which all the following is true:
+
+- **link text**: The element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
+- **non-link line text**: The element is [rendered on a line][] containing [visible][] [text nodes][text node] that are not [descendants][descendant] in the [flat tree][] of a [semantic link][].
+
+## Expectation
+
+Each target element has content (such as an image or text) visually located inside, or immediately before or after the test target, that indicates the test target is a link.
```

Comment:
Also, "visually located inside, or immediately before or after the test target" is not very good formulation.

* inside => do you mean inside the clientrect, the bounding box, the element (what does it mean to be visually inside an element?)
* **immediately** before or after => was does that mean for visual location? That between the image and the target there is only non-visible stuff? That the distance is small enough? That there is no other image or text between them?

Reply:
Tried to rephrase it to address this concern.